docs/src/pages/components/Icon.vue

```diff
@@ -5,7 +5,7 @@
         <p>A system icon, or UI icon, symbolizes a command, file, device, or directory. System icons are also used to represent common actions like trash, print, and save.</p>
         <p>You can see the full list of icons on the <a href="https://material.io/icons/" target="_blank" rel="noopener">Material Icons</a> website. Replace the spaces in the icon name by underscores (for example, <code>arrow back</code> becomes <code>arrow_back</code>).</p>
         <p>Alternatively you can use your PNG or SVG icons using the <code>md-src</code> property.</p>
-
+        <p>The use of font icons are suported too, although this approach lacks a few accessibility support.</p>
```

Comment:
"The use of font icons without ligatures are suported too". Because the Google Icons Font is "font icons" too. :)